src/components/FramelessViewer/FramelessViewerPageContainer.js

```diff
@@ -51,21 +47,12 @@ class FramelessViewerContainer extends Component {
       }).promise;
       this.setState({ parentFrameConnection });
     }
-    this.updateHeightWhenResize();
-  }
 
-  handleTextFieldChange(e) {
-    const fieldContents = JSON.parse(e.target.value);
-    trace(fieldContents);
-    const validated = validateSchema(fieldContents);
-    if (!validated) {
-      throw new Error(
-        "Certificate string does not conform to OpenCerts schema"
-      );
-    }
-    const verified = verifySignature(fieldContents);
-    trace(`Certificate verification: ${verified}`);
-    this.obfuscateDocument(fieldContents);
+    const config = { attributes: true, childList: true, subtree: true };
```

Comment:
Need this flag as well => https://developer.mozilla.org/en-US/docs/Web/API/MutationObserverInit/characterData

I did some test and when only the span text content change (for instance) then the observer dont trigger anything)